### DISCUSSION TITLE
Statusanzeige: NETZBETREIBER: ENTLADEN GESPERRT

### DISCUSSION THREAD

**Discussion by mmuegge** [solectrus/solectrus#3720] [General]
Auf der Anzeige "Strombilanz" in der Solectrus-Software erscheint nach dem Tausch des Akkus (LFP) durch Senec folgender Statustext:
NETZBETREIBER: ENTLADEN GESPERRT
Wahrscheinlich ist damit gemeint, das das Entladen gesperrt ist um eine Tiefentladung des Akkus zu verhindern.
Ich denke, das der Netzbetreiber damit nichts zu tun hat. 
Im Senec Webserver steht der Statustext "Entladen gesperrt"

**Comment by ledermann** [comment]
Ja, das ist ein bekanntes Problem, siehe hier:
https://github.com/orgs/solectrus/discussions/3282#discussioncomment-11186974

Lösung: Der SENEC-Collector muss einmalig neu gestartet werden, dann holt er sich die aktualisierten Status-Texte aus dem SENEC-Speicher.

Und wenn du gerade dabei bist: Es ist empfehlenswert, den SENEC-Collector aktuell zu halten, denn auch dafür gibt es regelmäßig Updates:
https://github.com/solectrus/senec-collector/releases

Insofern also: `docker compose pull && docker compose up -d`
